transform/i90_transform.py

```
@@ -91,107 +90,170 @@ def get_config_for_market(self, mercado: str) -> I90Config:
-        print("="*60)
+        try:
+            # Validate dataset_type
+            if dataset_type is None:
+                raise ValueError(f"dataset_type must be provided. Must be one of {self.dataset_types}")
+            else:
+                if dataset_type not in self.dataset_types:
+                    raise ValueError(f"Invalid dataset_type provided: {dataset_type}. Must be in {self.dataset_types}")
+
+            print(f"\n===== STARTING TRANSFORMATION RUN (Mode: {transform_type.upper()}) =====")
+            print(f"Dataset type to process: {dataset_type}")
+            if fecha_inicio: print(f"Start Date: {fecha_inicio}")
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)